Client/Frontend/Sync/SyncPairCameraViewController.swift

```diff
@@ -227,9 +227,7 @@ class SyncPairCameraViewController: SyncViewController {
     private func showErrorAlert(error: BraveSyncQRCodeError) {
         let errorMessage = { (error: BraveSyncQRCodeError) -> String in
             switch error {
-            case .insecure: return Strings.syncInsecureVersionError
```

Comment:
`insecure` and `newerVersion` aren't the same.
`Insecure` = When you scan QRCode v1
`NewVersion` = When you scan a version that is higher than your current browser supports. `"This sync code was generated by an outdated version of Brave on another device."` wouldn't work.

Reply:
good point, we will update the ticket with new copy and I will change it, @AlexeyBarabash to update it

Reply:
Updated at https://github.com/brave/brave-ios/issues/4908#issue-1116614816